src/multiplexer/EventHandler.cpp

```diff
@@ -140,12 +140,24 @@ void EventHandler::handleNewConnection(Server & s) {
 }
 
 bool EventHandler::isResponseComplete(int clientFd) {
+	// check header completion
 	std::string buff = _clients[clientFd]->_requestBuffer;
 	size_t pos = buff.find("\r\n\r\n");
-	if (pos == std::string::npos)
-		return false;
+	if (pos == std::string::npos) {
+		pos = buff.find("\n\n");
+		if (pos == std::string::npos) {
+			return false;
+		}
+	}
 
+	if (isHeaderChunked(clientFd)) {
+		_clients[clientFd]->_reqType = (ClientConnection::reqType)CHUNKED;
```

Comment:
not a fan of this, but it was a quick fix for an error. Ideally I wouldn't repeat the enum in both classes.